File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
// 250723

class Solution {
    public String solution(String my_string) {
        String answer = "";
        
        for(int i=0; i<my_string.length(); i++) {
            char ch = my_string.charAt(i);
            if ('a' <= ch && ch <= 'z') answer += (char)('A' + (ch-'a')); // 소문자->대문자
            else answer += (char)('a' + (ch-'A')); // 대문자->소문자
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.py
# 251122

def solution(my_string):
    answer = ''
    
    for i in range(len(my_string)):
        if 'A'<=my_string[i]<='Z':
            answer += my_string[i].lower()
        else:
            answer += my_string[i].upper()
    
    return answer